Dockerfile

```diff
@@ -1,34 +1,33 @@
-FROM node:12-stretch
-LABEL description="Debian image to build nativefier apps"
+FROM node:12-alpine
```

Comment:
Happy to switch to Alpine. In the past it used to have binary compatibility (because musl, I think?) trouble.

Did you test the final image under Windows and macOS/Linux? Do you confirm it runs fine?

Reply:
I did test the final generated app under Windows and Mac. I could install VirtualBox and test under Linux if needed as well.